docs/validators-and-full-nodes/run-full-node-mainnet.md

```diff
@@ -65,12 +112,14 @@ secretd configure-secret node-master-cert.der "$SEED"
 
 perl -i -pe 's/^seeds = ".*?"/seeds = "bee0edb320d50c839349224b9be1575ca4e67948\@secret-2.node.enigma.co:26656"/' ~/.secretd/config/config.toml
 perl -i -pe 's;laddr = "tcp://127.0.0.1:26657";laddr = "tcp://0.0.0.0:26657";' ~/.secretd/config/config.toml
+```
 
+#### Start your node as a service:
+
+```bash
 sudo systemctl enable secret-node
 
 sudo systemctl start secret-node # (Now your new node is live and catching up)
-
-secretcli config node tcp://localhost:26657
```

Comment:
Why did you removed this line here?

Reply:
I moved it down below the `journalctl` and added some more info after (looking at the `status` and what indicates it's fully sync'd).

Reply:
Yeah I saw that, I just thought it made more sense in the original place. Doesn't really matter :)